package.json

```diff
@@ -3,11 +3,13 @@
   "version": "3.0.0",
   "description": "Simple i18n translation and localization components and helpers for React applications.",
   "main": "./build/index.js",
+  "types": "types",
   "scripts": {
     "test": "jest --verbose",
     "test:watch": "npm test -- --watch",
     "lint": "eslint src/**/*.jsx src/**/*.js __test__/**/*.js",
     "lint:fix": "npm run lint -- --fix",
+    "dtslint": "dtslint types",
```

Comment:
I intentionally didn't add this to the precommit because it takes ~10s to run, which make committing pretty annoying.